### DISCUSSION TITLE
Subscribe to multiple collections using Changes API

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#18249] [Q&A] [answered]
I need to subscribe to multiple collections using the Changes API. It does not seem to be possible but maybe you could add or have a solution for doing so?

**Comment by ayende** [comment, marked as answer]
```
var changes = store.Changes();
changes.ForDocumentsInCollection<User>().Subscribe(c => /* */);
changes.ForDocumentsInCollection<Dogs>().Subscribe(c => /* */);
```

This will create  a _single_ web socket request to the server, for both of those.
